tools/benchcomp/benchcomp/parsers/kani_perf.py

```diff
@@ -0,0 +1,80 @@
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+# Read CBMC-printed statistics from the kani perf regression suite
```

Comment:
Now that I understand this better, the statistics we're reading from `expected.out` files actually include output from Kani (e.g. the `Verification Time` line). So, perhaps this should say "Read Kani and CBMC statistics" or just "Extract statistics from Kani output".